index.html

```diff
@@ -9,14 +9,9 @@
     content="Interactive playground for experimenting with Slang shading language. Write, compile, and visualize shader code in real-time." />
   <meta name="keywords" content="Slang, shader, WebGPU, graphics programming, code playground, real-time rendering" />
   <link href="./styles/styles.css" rel="stylesheet" />
-  <link href="json-tree/jsontree.js.css" rel="stylesheet" />
+  <link rel="stylesheet" href="node_modules/jjsontree.js/dist/jsontree.js.css">
```

Comment:
I believe we modified the jaontree css file a bit to hide the title bar.

Reply:
ah, ok. Will look into it once I get it working.